transit_odp/fares/tasks.py

```diff
@@ -172,6 +174,27 @@ def task_run_fares_validation(task_id):
     revision.save()
 
 
+@shared_task
+def task_set_fares_validation_result(task_id):
+    """Task to set validation errors in a fares file/s."""
+    task = get_etl_task_or_pipeline_exception(task_id)
+    revision = task.revision
+    context = DatasetPipelineLoggerContext(object_id=revision.dataset.id)
+    adapter = PipelineAdapter(logger, {"context": context})
+
+    file_ = revision.upload_file
+
+    org_id_list = Dataset.objects.filter(id=revision.dataset_id).values_list(
+        "organisation_id", flat=True
+    )
+
+    adapter.info("Setting validation errors.")
+    fares_validator_obj = FaresXmlValidator(file_, org_id_list[0], revision.id)
+    fares_validator_obj.set_errors()
+
+    task.update_progress(50)
+
```

Comment:
Please add some logs saying that fares validation started and ended.

Reply:
Done this change